app/backend/core/imageshelper.py

```diff
@@ -34,3 +38,49 @@ async def fetch_image(blob_container_client: ContainerClient, result: Document)
         else:
             return None
     return None
+
+
+def get_image_dims(image):
```

Comment:
These functions are taken from https://github.com/openai/openai-cookbook/pull/881/files

Reply:
do you mind adding this a s a comment?